packages/apps/tools/src/components/Global/HoverTag/HoverTag.tsx

```diff
@@ -0,0 +1,45 @@
+import type { IMaskedValueProps } from '@kadena/react-ui';
+import { SystemIcon, Tag, Tooltip, maskValue } from '@kadena/react-ui';
+import React, { useRef } from 'react';
+import { copyButtonClass, monoFontTextStyles } from './HoverTag.css';
+
+export interface IHoverTagProps {
+  value: IMaskedValueProps['value'];
+}
+
+export const HoverTag = ({ value }: IHoverTagProps) => {
+  const [tooltipContent, setTooltipContent] = React.useState<string>(value);
+  const ref = useRef<HTMLDivElement>(null);
+  return (
+    <>
+      <span
+        data-testid="kda-hover-tag"
+        onMouseEnter={(e: React.MouseEvent<HTMLElement>) =>
+          Tooltip.handler(e, ref)
+        }
+        onMouseLeave={(e: React.MouseEvent<HTMLButtonElement>) =>
+          Tooltip.handler(e, ref)
+        }
+      >
+        <Tag>
+          <span className={monoFontTextStyles}>{maskValue(value)}</span>
```

Comment:
you can use the `Text` component with font set to "mono" for this as well

Reply:
Ah super, did that, thank you!